pkg/client/graphql.go

```diff
@@ -8,52 +8,62 @@ import (
 	"strings"
 )
 
+const (
+	constEvents              = `events`
+	constEventReceivers      = `event_receivers`
+	constEventReceiverGroups = `event_receiver_groups`
+	constEvent               = `event`
+	constEventReceiver       = `event_receiver`
+	constEventReceiverGroup  = `event_receiver_group`
+)
```

Comment:
I suggest naming these to capture their use, and I'd also suggest removing the "const" prefix. If I'm understanding their usage, my proposal would be "eventsQuery", "eventMutation", etc.